packages/langchain/src/openai/types.ts

```diff
@@ -35,6 +35,18 @@ export type AzureOpenAiChatModelParams = Pick<
    * If `undefined` the `strict` argument will not be passed to OpenAI.
    */
   supportsStrictToolCalling?: boolean;
+  /**
+   * Whether the model should automatically stream responses when using `invoke()`.
+   * If {@link disableStreaming} is set to `true`, this option will be ignored.
+   * If {@link streaming} is explicitly set to `false`, {@link disableStreaming} will be set to `true`.
+   */
+  streaming?: boolean;
+  /**
+   * Whether streaming should be disabled.
+   * If `true`, `stream()` calls will be redirected to `invoke()`.
+   * Setting `streaming: false` will automatically set this to `true`.
+   */
+  disableStreaming?: boolean;
```

Comment:
[req] Do you still need this since it is part of `BaseChatModelParams` which is unioned in the next line?

https://github.com/langchain-ai/langchainjs/blob/cc502e1b67dbadcd123a7ea2964c791c2bbad581/libs/langchain-core/src/language_models/chat_models.ts#L101